src/main/java/com/searchservice/app/rest/ManageTableResource.java

```diff
@@ -106,7 +106,7 @@ public ResponseEntity<TableSchemav2> getTable(@RequestParam int tenantId, @PathV
 				tableInfoResponseDTO.setMessage("Table Information retrieved successfully");
 				return ResponseEntity.status(HttpStatus.OK).body(tableInfoResponseDTO);
 			} else {
-
+                 tableInfoResponseDTO.setMessage("Table not found");
 				throw new BadRequestOccurredException(HttpStatusCode.BAD_REQUEST_EXCEPTION.getCode()
 						, "REST operation couldn't be performed");
```

Comment:
Can you pls change this response message also, with something relatable with the method call?